product_variant_weight/hooks.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def post_init_hook(cr, registry):
+    """This post-init-hook will update all existing product.product weight"""
+    cr.execute("""UPDATE product_product pp
```

Comment:
Make 2 SQL (one per field) and put the condition `WHERE <field> is NULL` to avoid overwriting previous values (although it's not the usual, maybe you have loaded previously the values, or you have made a cycle of installing/uninstalling the module without dropping the columns).